docs/getting-started/integrations-cdn/react-cdn.md

```diff
@@ -0,0 +1,578 @@
+---
+menu-title: Default integration
+meta-title: React rich text editor component via CDN | CKEditor 5 documentation
+meta-description: Install, integrate and configure CKEditor 5 using the React component with CDN.
+category: react-cdn
+order: 10
+---
+
+{@snippet installation/integrations/framework-integration}
+
+# React rich text editor component via CDN
+
+<p>
+	<a href="https://www.npmjs.com/package/@ckeditor/ckeditor5-react" target="_blank" rel="noopener">
+		<img src="https://badge.fury.io/js/%40ckeditor%2Fckeditor5-react.svg" alt="npm version" loading="lazy">
+	</a>
+</p>
+
+This guide explains how to integrate CKEditor5 into your React application using CDN.
+
+<info-box hint>
+	Starting from version 6.0.0 of this package, you can use native type definitions provided by CKEditor&nbsp;5. Check the details about {@link getting-started/setup/typescript-support TypeScript support}.
+</info-box>
+
+## Quick start with CKEditor 5 React and CDN
+
+### Using CKEditor&nbsp;5 Builder
+
+The easiest way to use CKEditor&nbsp;5 in your React application is by configuring it with the [CKEditor&nbsp;5 Builder](https://ckeditor.com/builder?redirect=docs) and integrating it with your application. The Builder offers an easy-to-use user interface to help you configure, preview, and download the editor suited to your needs. You can easily select:
+
+* the features you need,
+* the preferred framework (React, Angular, Vue or Vanilla JS),
+* the preferred distribution method.
+
+You get ready-to-use code tailored to your needs!
+
+### Setting up the project
+
+This guide assumes you have a ready React project. If not, you can create a basic React project using [Vite](https://vitejs.dev/). Refer to the [React documentation](https://react.dev/learn/start-a-new-react-project) to learn how to set up a project in the framework.
+
+### Installing from npm
```

Comment:
This is confusing, we are in a CDN guide after all.

Reply:
Yes indeed, it is a CDN guide, but to use CDN with react you still need to download `@ckeditor/ckeditor5-react` from npm.

Reply:
I realize that, but maybe we can mention it? Or is it just me and it's clear?